src/cpp/src/llm_pipeline.cpp

```diff
@@ -22,80 +22,6 @@ bool with_cpu_sve();
 
```

Comment:
I suppose we need to move forward declaration for `with_cpu_sve();` to utils.cpp as well